Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/library/rust:latest as builder
+
+WORKDIR /app
+COPY . .
+RUN cargo build --release
+
+FROM gcr.io/distroless/cc-debian12:nonroot AS runtime
+COPY --from=builder /app/target/release/imdl /imdl
```

Comment:
Should `idml` be copied instead to `/user/local/bin` since that's the appropriate path for user-installed binaries?

Reply:
@casey I've changed the path on https://github.com/casey/intermodal/pull/533/commits/53937e534215625f1ab310512631ee68b4e05e44